lib/Backend/BackwardPass.cpp

```diff
@@ -1818,6 +1908,15 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr)
             BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed = byteCodeUsesInstr->byteCodeUpwardExposedUsed;
             if (byteCodeUpwardExposedUsed != nullptr)
             {
+                //Stack Args for Formals optimization.
```

Comment:
Seems like you already do this in ProcessSymUse...  So why is it still needed here?

---
In reply to: [68475602](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68475602) [](ancestors = 68475602,67800813,67740116)